### PR TITLE
Compatibility with pstoedit 3.74 and gs 9.27

### PR DESCRIPTION
**Pull request by jcwinkler**
Related issue: #126 

Precise description of the changes proposed in the pull request: Make TexText compatible with pstoedit 3.74 and gs 9.27. Some harsh adjustments in the tests have been necessary, however, after manually checking the results it looks really good.

Short checklist:
- [x] Tested with Inkscape version: 0.92.3, 0.92.4
- [x] Tested on Linux, Distro: Kubuntu 18.04
- [x] Tested on Windows 8.1 x64
- [ ] Tested on MacOS, Version:  [fill in MacOS version/ name here]

### CODE REVIEW

File: pytests/test_compatibility.py [hunk 1]
```diff
@@ -10,16 +10,18 @@
...
     def __enter__(self):
-        self.__name = tempfile.mkdtemp()
+        self.__name = tempfile.mkdtemp(suffix=self.__suffix)
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         if self.__name is not None:
-            shutil.rmtree(self.__name)
+            pass
+            #shutil.rmtree(self.__name)
```
Comment (sizmailov) [inline pytests/test_compatibility.py]: Remove debugging stuff.

File: pytests/test_compatibility.py [hunk 2]
```diff
@@ -10,16 +10,18 @@
 
 
 class TempDirectory(object):
-    def __init__(self):
+    def __init__(self, suffix=""):
```
Comment (sizmailov) [inline pytests/test_compatibility.py]: This change feels wrong. Temp dir should be ... temporal, so the name does not matter. If you want to distinguish between test in debug - create root temp directory with named directories inside. While this ad-hoc debug technique might be helpful, I think it should not be merged.

**Comment by sizmailov** [comment]
So pstoedit works with gs?! Great!

File: pytests/test_compatibility.py (hunk 2 above)
Comment (jcwinkler) [reply pytests/test_compatibility.py]: I changed it such that one can decide whether to have temporary files in the system's temp dir (i.e. the old behavior) or to keep the results in a separate folder (for manual inspection after the tests).

Comment (sizmailov) [reply pytests/test_compatibility.py]: It resides in tests only, so let it be.

Comment (sizmailov) [reply pytests/test_compatibility.py]: Just a thought. We have TempDir guard in main extension code too. I think switching temp dir to known place might be helpful feature in debugging on user's side, note #125.

**Pull request merged**